django/core/checks/security/base.py

```diff
@@ -113,6 +113,11 @@
     id='security.W022',
 )
 
+W023 = Warning(
```

Comment:
This should be `W024` (since there is already an `E023`).

Also it feels like this should be an error not a warning. If the check fails we will get an error (in behaviour) somewhere down the line. It would also be consistent with other checks similar to this.